cmd/podman/common/build.go

```diff
@@ -453,6 +454,12 @@ func buildFlagsWrapperToOptions(c *cobra.Command, contextDir string, flags *Buil
 		runtimeFlags = append(runtimeFlags, "--systemd-cgroup")
 	}
 
+	configIndex := libpod.RuntimeNameToConfigIndex(podmanConfig.RuntimePath)
```

Comment:
you cannot import libpod into the remote client, this function would need to live elsewhere but really I would say just inline this